README.md

```diff
@@ -69,7 +69,7 @@ agent = AnyAgent.create(
     tracing=TracingConfig(output_dir="traces") # Optional, but recommended for saving and viewing traces
 )
 
-agent.run("Which Agent Framework is the best??")
+result: AgentResult = agent.run("Which Agent Framework is the best??")
```

Comment:
I added the typing not only for readability but also because then our CI tests will throw an error if we change the import location or name of the result class "which we discuss in the text part of the docs"

Reply:
um. but do you think we should put it in the README? it feels like just more verbose

Reply:
I'm fine with removing it to keep the README simple